### PR TITLE
update NOTICE

### PR DESCRIPTION
**Pull request by MP91**
## Describe your changes
Updated the Notice file with the reference to the SBOM of the base-image

### CODE REVIEW

**Review by doosuu — APPROVED** [review]
LGTM

File: NOTICE.md [hunk 1]
```diff
@@ -17,55 +15,19 @@ SPDX-License-Identifier: Apache-2.0
...
-| Brew                | https://github.com/Homebrew/brew/blob/master/LICENSE.txt                                                         |
 | K9S                 | https://github.com/derailed/k9s/blob/master/LICENSE                                                              |
 | Redis               | https://redis.com/legal/licenses/                                                                                |
 | Mosquitto           | https://github.com/eclipse/mosquitto/blob/master/LICENSE.txt                                                                                                                 |
 
 ### Development
 | Software |                               License                                |
 | :------: | :------------------------------------------------------------------: |
-|  Thread  | https://github.com/loretoparisi/python-threading/blob/master/LICENSE |
 |   GRPC   |           https://github.com/grpc/grpc/blob/master/LICENSE           |
```
Comment (kse3hi) [inline NOTICE.md:29]: Is grpc and grpcio the same things?

File: NOTICE.md [hunk 2]
```diff
@@ -17,55 +15,19 @@ SPDX-License-Identifier: Apache-2.0
 | Software            | License                                                                                                          |
 | :------------------ | :--------------------------------------------------------------------------------------------------------------- |
 | Dapr                | https://github.com/dapr/dapr/blob/master/LICENSE                                                                 |
```
[the remark this answers is not in the document]
Comment (MP91) [reply NOTICE.md:17]: This file can be seen as a general overview of the main tools we use. More details can than be found in the 3rd party notice

File: NOTICE.md (hunk 1 above)
Comment (MP91) [reply NOTICE.md:29]: I would say GRPC is the general idea of communication and grpcio is one implementation of it.

Comment (kse3hi) [reply NOTICE.md:29]: From the license point of view - they are the same things

**Pull request merged**